src/Ocelot/Configuration/HttpHandlerOptions.cs

```diff
@@ -6,11 +6,12 @@
     /// </summary>
     public class HttpHandlerOptions
     {
-        public HttpHandlerOptions(bool allowAutoRedirect, bool useCookieContainer, bool useTracing)
+        public HttpHandlerOptions(bool allowAutoRedirect, bool useCookieContainer, bool useTracing, bool useProxy = true)

```

Comment:
Not sure that we need this to default here as we default it in the FileConfiguration but I dont have a big problem with it

Reply:
I would have also liked to avoid this but the main reason to default it here as well was to make the parameter optional and that way keep the change as small as possible, but I am happy to remove it from here and update all the references (mainly tests, about 25 references) if needed.

Reply:
@marcoaam if you are happy to do it that would be awesome! I don't mind merging and doing it myself if you don't have the time! Let me know what you prefer.

Reply:
Thanks for the quick response. No worries @TomPallister, It should only take a minute. I will get that done now.